application/src/main/java/org/opentripplanner/updater/vehicle_rental/datasources/params/AllowedRentalType.java

```diff
@@ -0,0 +1,32 @@
+package org.opentripplanner.updater.vehicle_rental.datasources.params;
+
+import org.opentripplanner.framework.doc.DocumentedEnum;
+
+/**
+ * This is temporary and will be removed in a future version of OTP.
+ *
+ * Enum to specify the type of rental data that is allowed to be read from the data source.
+ */
+public enum AllowedRentalType implements DocumentedEnum<AllowedRentalType> {
```

Comment:
This class should be renamed to `RentalPickupType` and have ´{STATION, FREE_FLOTING}´ as values, not `ALL`.

@leonardehrenfried do you agree? I changed the VEHICLES to FREE_FLOTING. Also use singular, not plural in enums.

In places where there is a set of these you should use `Set<RentalPickupType>`. I would define a constant in the enum class for ALL like this:

```Java
private static final Set<RentalPickupType> ALL = Collections.unmodifiableSet(EnumSet.allOf(RentalPickupType.class));
```
The `unmodifiableSet(...)` is needed to avoid modification and you should use it instead of `Set.copyOf()` (witch convert the internal set to a `HashSet`).

Reply:
I agree.